src/cmd-generate-release-meta

```diff
@@ -138,7 +138,7 @@ def append_build(out, input_):
                 }
             }
     # Aliyun/AWS specific additions
-    for meta_key, cloud, image_field in ("aliyun", "aliyun", "id"), ("amis", "aws", "hvm"):
+    for meta_key, cloud, image_field in ("aliyun", "aliyun", "id"), ("amis", "aws", "hvm"), ("ibmcloud", "ibmcloud", "image"), ("powervs", "powervs", "image"):
```

Comment:
Do you mind showing me an example of what this looks like when run? Is it an image name or a URL? If it's a URL maybe we should be representing this a different way.

Reply:
actually i realize that we need the bucket name to key off of also. i added that for IBMCloud/PowerVS. this is how the release meta looks:
```
"powervs": {
          "artifacts": {
            "ova": {
              "disk": {
                "location": "https://builds.coreos.fedoraproject.org/prod/streams/master/builds/410.84.202109171742-0/ppc64le/rhcos-410.84.202109171742-0-powervs.ppc64le.ova",
                "signature": "https://builds.coreos.fedoraproject.org/prod/streams/master/builds/410.84.202109171742-0/ppc64le/rhcos-410.84.202109171742-0-powervs.ppc64le.ova.sig",
                "sha256": "8b8ac96079aade6f813c3d4d33b6042920825b1e622b799d5e7a28fd515e1d87",
                "uncompressed-sha256": null
              }
            }
          },
          "images": {
            "us-east": {
              "image": "rhcos-410-84-202109171742-0-powervs-ppc64le.ova",
              "bucket": "rhcos-powervs-images-us-east"
            },
            "eu-de": {
              "image": "rhcos-410-84-202109171742-0-powervs-ppc64le.ova",
              "bucket": "rhcos-powervs-images-eu-de"
            },
            "us-south": {
              "image": "rhcos-410-84-202109171742-0-powervs-ppc64le.ova",
              "bucket": "rhcos-powervs-images-us-south"
            }
          }
        },
```
Also this is before i added your suggestion below where the platform and the arch have swapped places.

Reply:
@clnperez this is how the release metadata will look in the openshift installer. I take it that this information is sufficient for your needs ?

Reply:
here's my complaint here. we're not actually creating an image are we? we're just uploading a file into object storage.

As such it would seem like we should be using the `url` key here instead of `image`. Maybe like is done here for azure: https://github.com/openshift/installer/blob/f6ece2a15c0d0fd57028be53df619cf48acefa11/data/data/rhcos-stream.json#L564

Reply:
unfortunately power vs image import doesn't understand the url format, it understands only the bucket, region and object name but having that said, if we can include url also with less effort I'm okay(
may be helpful someone if they want to download this image and use them in their own bucket or so) but this format looks promising to me atm.